src/web/app/pages-help/instructor-help-page/instructor-help-sessions-section/instructor-help-sessions-section.component.html

```diff
@@ -409,6 +435,42 @@ <h4 *ngIf="displaySubsection(4, 7)">Managing Session Responses</h4>
     </tm-example-box>
   </tm-instructor-help-panel>
   <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.STUDENTS_SEE_RESPONSE)"
+                            [id]="SessionsSectionQuestions.STUDENTS_SEE_RESPONSE"
+                            headerText="When do students see responses?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.STUDENTS_SEE_RESPONSE]">
+    <p>
+      Responses are available to view after you have <em>published</em> the session results. Furthermore, which information the students can see is determined by the <em>visibility</em> settings of each question.
+    </p>
+  </tm-instructor-help-panel>
+  <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.EDIT_AFTER_SUBMIT)"
+                            [id]="SessionsSectionQuestions.EDIT_AFTER_SUBMIT"
+                            headerText="Can students edit responses after they have submitted them?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.EDIT_AFTER_SUBMIT]">
+    <p>
+      Students can edit responses any number of times, as long as the session is open for submissions.
+    </p>
+  </tm-instructor-help-panel>
+  <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.EXTEND_DEADLINE)"
+                            [id]="SessionsSectionQuestions.EXTEND_DEADLINE"
+                            headerText="Can I extend a sessions deadline?"
+                            [(isPanelExpanded)]="questionsToCollapsed[SessionsSectionQuestions.EXTEND_DEADLINE]">
+    <p>
+      Yes, it is possible to change the deadline of a session simply by editing the end date/time of it. But note that the deadline extension applies to all students in the course. Note that changes to a deadline can cause certain automated emails to be sent again, if applicable. For example, if you extend the deadline of a session that has already been closed, the 'session closing soon' email will be sent again around 24 hours before the new deadline.
+    </p>
+  </tm-instructor-help-panel>
+  <!--Question-->
+  <tm-instructor-help-panel #question [hidden]="key && !showQuestion.includes(SessionsSectionQuestions.SPECIFIC_STUDENT_EXTENSION)"
+                            [id]="SessionsSectionQuestions.SPECIFIC_STUDENT_EXTENSION"
+                            headerText="Can I extend the deadline for a specific student?">
+
+    <p>
+      At the moment it is not possible to extend the deadline for a specific student, but you can get the student to send their answers to you  so that you can submit it on their behalf, using the 'moderate' feature.
```

Comment:
```suggestion
      At the moment it is not possible to extend the deadline for a specific student, but you can get the student
      to send their answers to you  so that you can submit it on their behalf, using the 'moderate' feature.
```